<launch>

  <node pkg="heuristic_grids" type="heuristic_grids_server" name="heuristic_grids_server"
    output="screen">

    <!--  [m] Size of grid squares -->
    <param name="grid_square_size" type="double" value="0.02"/>

    <!-- [m] Minimum distance should probably be at least the robot radius (0.13) -->
    <param name="min_distance" type="double" value="0.13"/>

    <!--  Size of Gaussian smoothing kernel, should be odd number and about
    3 times the size of the standard deviation-->
    <param name="smoothing_kernel_size" type="int" value="21"/>

    <!--  Standard devoiation of the Gaussian smoothing kernel -->
    <param name="smoothing_kernel_sd" type="int" value="7"/>

  </node>

</launch>
